packages/grafana-ui/src/components/ButtonCascader/ButtonCascader.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+
+// @ts-ignore
+import RCCascader from 'rc-cascader';
+
+export interface ButtonCascaderOption {
+  label: string;
+  value: string;
+
+  children?: ButtonCascaderOption[];
+  disabled?: boolean;
+  // Undocumented tooltip API
+  title?: string;
+}
+
+export interface ButtonCascaderProps {
+  options: ButtonCascaderOption[];
+  buttonText: string;
```

Comment:
Instead of property, maybe we can use `children` property as `Button` component does?

Reply:
Maybe this sould be done in a seperate PR along with some other ButtonCascader changes